packages/core/src/app.ts

```diff
@@ -620,7 +620,7 @@ export class AppBuilder<TreeNames> {
       .forEach(a => a.differ.update());
     return this.appDiffers.map(a => {
       return Object.assign({}, a.engine, {
-        appFiles: new AppFiles(a.differ, this.resolvableExtensionsPattern),
+        appFiles: new AppFiles(a.differ, this.resolvableExtensionsPattern, this.options.podModulePrefix),
```

Comment:
Indeed, that's great! I didn't notice as that method was only defined in the compat package, and not part of the `AppAdapter` interface here.

Reply:
Oh yeah, then we will need to add it to that interface.

Reply:
The boundary between core and compat is pretty strong for addons, but it's not for the app yet, because we don't have a second kind of app yet. Don't worry about it too much right now.